src/app/zap-templates/zcl/data-model/chip/mode-select-cluster.xml

```diff
@@ -17,9 +17,30 @@ limitations under the License.
 <configurator>
   <domain name="CHIP"/>
 
+  <enum name="ChangeToModeResponseStatus" type="ENUM8">
+    <cluster code="0x0050"/>
+    <item value="0x0" name="Success"/>
+    <item value="0x1" name="UnsupportedMode"/>
+    <item value="0x2" name="GenericFailure"/>
```

Comment:
As described below for SemanticTags, this is not the full set of possible response statuses, which is a bit of a complication....

Reply:
Similar argument below. Again, happy to remove it.